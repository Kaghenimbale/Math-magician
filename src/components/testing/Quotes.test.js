import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Quote from '../Quotes';

describe('Quote componenet', () => {
  test('should render the Quote component', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('should have "Let us do some Math" ', () => {
     render(<Quote />);
  const HomeElement = screen.getByTestId('Quote');
  expect(HomeElement).toBeInTheDocument();
  });
});