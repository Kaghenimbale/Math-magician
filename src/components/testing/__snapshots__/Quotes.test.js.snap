// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote componenet should render the Quote component 1`] = `
<div
  className="Content content-2"
>
  <div
    className="quotes"
    data-testid="Quote"
  >
    <div
      className="loading"
      data-testid="Loading"
    >
      <img
        alt="loading-icon"
        className="load"
        src="Loading_icon.gif"
      />
    </div>
  </div>
</div>
`;
